src/main/scala/tech/cryptonomic/conseil/config/Shared.scala

```diff
@@ -63,6 +64,15 @@ final case class SodiumConfiguration(libraryPath: String) extends AnyVal with Pr
 /** holds configuration for the akka-http-caching used in metadata endpoint */
 final case class HttpCacheConfiguration(cacheConfig: com.typesafe.config.Config)
 
+/** configuration for fetching baking and endorsing rights */
+final case class Rights(
```

Comment:
Personal stance: I might suggest we try to think of a more specific name for this configuration.

It sounds very generic like this and easy to get confused with other rights-related concept

Reply:
renamed